server/npg/porchdb/data_access.py

```diff
@@ -38,25 +38,25 @@ def __init__(self, session):
         self.session = session
 
     async def get_pipeline(self, pipeline: Pipeline):
```

Comment:
Since we put a Pipeline in and get a Pipeline out, I think we need a description here to explain why it exists (i.e. to in-fill a partially described pipeline). Maybe it could also have a type hint `-> Pipeline` in the signature